registries/intranet_agents_with_tools.hocon

```diff
@@ -21,53 +33,10 @@
             Only answer inquiries that are directly within your area of expertise, from the company's perspective.
             Do not try to help for personal matters.
             Do not mention what you can NOT do. Only mention what you can do.
+            Alwasy ask your down-chain agents for their take on the inquiry, and delegate to them as much as possible
+            to fulfill the user's inquiry.
+            If needed ask the user for clarification.
             """,
-            "aaosa_instructions": """
```

Comment:
These definitions are not needed anymore: they are now imported from `aaosa.hocon`